pallets/elections/src/impls.rs

```diff
@@ -329,7 +329,8 @@ where
         Self::adjust_rewards_for_session();
 
         // clear block count
-        SessionValidatorBlockCount::<T>::remove_all(None);
+        let result = SessionValidatorBlockCount::<T>::clear(u32::MAX, None);
```

Comment:
is `MAX` safe? I mean, maybe we should try to ensure that we will fill in the block or whatever... or maybe it is checked internally?

Reply:
it will clear up to that amount of entries in the map. So max cleared items is the size of reserved + non_reserved